scripts/plot_storage_levels.py

```diff
@@ -19,8 +19,8 @@
 -------------
 The script creates storage level plots.
 
-The static plots are saved as files (png, pdf, etc.) and the interactive plotly plots as html-files
-in a new directory called plotted.
+The static plots are saved as png-files and the interactive plotly plots as html-files
```

Comment:
Same as before. The file extension is defined in config.settings.general.plot_filetype

Reply:
See above. Also done in other docstrings of scripts importing `plot_filetype`.